recipe/meta.yaml

```diff
@@ -1,14 +1,18 @@
-{% set major_minor = "2.38" %}
-{% set version = major_minor + ".1" %}
+{% set version = "2.41" %}
```

Comment:
Not sure why the `.1` patch version used to be added here. I think it's better to stick with the upstream versioning.

Reply:
seems for a first look that they just included the OSX patch below.